### PR TITLE
Added API keys API

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/__tests__/resources/apiKeys.integration.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Cognite AS
+
+import { API } from '../../resources/api';
+import { NewApiKeyResponseDTO, ServiceAccount } from '../../types/types';
```
Comment (polomani) [inline src/__tests__/resources/apiKeys.integration.spec.ts]: I believe we don't put `DTO` in the end of DTOs explicitly

File: src/standardMethods.ts [hunk 2]
```diff
@@ -127,6 +127,29 @@ export function generateListNoCursorEndpoint<ResponseType>(
...
+        method: 'get',
+        url: resourcePath,
+        params,
+      },
+      true
+    );
+    return metadataMap.addAndReturn(response.data.items, response);
+  };
+}
+
```
Comment (polomani) [inline src/standardMethods.ts:152]: you have the same method but with no params
why not making params optional?
problems dynamic typing?

File: src/types/types.ts [hunk 3]
```diff
@@ -1018,6 +1018,36 @@ export interface AssetMapping3DBase {
...
+  id: CogniteInternalId;
+  serviceAccountId: CogniteInternalId;
+  createdTime: Date;
+  status: 'ACTIVE' | 'DELETED';
+}
+
+export interface ApiKeyListScope {
+  all?: boolean;
+  serviceAccountId?: CogniteInternalId;
+  includeDeleted?: boolean;
```
Comment (polomani) [inline src/types/types.ts:1155]: it has some explanation in spec, why didn't add it here?

File: src/standardMethods.ts (hunk 2 above)
Comment (ghost) [reply src/standardMethods.ts:152]: Yes, dynamic typing.. 🤔 Haven't found a good way to do it

**Review by polomani — APPROVED** [review]

**Pull request merged**